.github/workflows/workflow.yml

```diff
@@ -291,4 +273,4 @@ jobs:
           client-secret: ${{ secrets.eg_client_secret }}
           access-token: ${{ secrets.eg_access_token }}
           type: "cpcodes"
-          ref: ${{ format('{0},{1}', secrets.ns_cdn_cpcode, secrets.prop_cpcode) }}
+          ref: ${{ format('{0},{1},{2}', secrets.ns_cpcode, secrets.ns_cdn_cpcode, secrets.prop_cpcode) }}
```

Comment:
This cache clear needs all three CP codes in it since we removed the earlier one. I have also fixed the Akamai permissions issue.